### PR TITLE
Updating to remove unsupported getRecord with layout value.

### PR DESCRIPTION
**Pull request by dbreese**
Instead, it is currently recommended to not use layouts and to specify field values for mobile clients due to offline edge cases.

@khawkins @AdamLiechty

### CODE REVIEW

**Comment by dbreese** [comment]
Lint error I didnt see, looking to find out how to fix it:

```
npm run lint

> offline-starter-kit@1.0.0 lint /Users/dbreese/dev/offline-app-developer-starter-kit
> eslint "force-app/**/*.js"


/Users/dbreese/dev/offline-app-developer-starter-kit/force-app/main/default/lwc/viewAccountRecord/viewAccountRecord.js
  15:53  error  'fields' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)
```

**Comment by dbreese** [comment]
Hmmm, tests fail using Node 16, locally too. With: 

```
 RUNS  force-app/main/default/lwc/viewAccountRecord/__tests__/viewAccountRecord.test.js
/Users/dbreese/dev/offline-app-developer-starter-kit/force-app/main/default/lwc/viewAccountRecord/viewAccountRecord.js:168
          fields: this.fields,
                       ^

TypeError: Cannot read properties of undefined (reading 'fields')
    at config (/Users/dbreese/dev/offline-app-developer-starter-kit/force-app/main/default/lwc/viewAccountRecord/viewAccountRecord.js:168:24)
    at /Users/dbreese/dev/offline-app-developer-starter-kit/node_modules/@lwc/engine-dom/dist/engine-dom.cjs.js:6283:31
    at ReactiveObserver.observe (/Users/dbreese/dev/offline-app-developer-starter-kit/node_modules/@lwc/engine-dom/dist/engine-dom.cjs.js:661:13)
    at computeConfigAndUpdate (/Users/dbreese/dev/offline-app-developer-starter-kit/node_modules/@lwc/engine-dom/dist/engine-dom.cjs.js:6283:8)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by dbreese** [comment]
Soooo, referencing a field in the @wire cause some odd issues... it must reference via `this.` for lint to pass... however, node 14 tests will run, but fail in node 16. I think I will need to update PR to hard-code the list of fields in the @wire

File: README.md [hunk 1]
```diff
@@ -36,13 +36,19 @@ The Offline App Developer Starter Kit provides instructions and sample code to g
...
    ```
 
-3. Authorize access to your org. Either Salesforce CLI or VS Code can be used for authorization and deployment. VS Code is recommended.
+1. Install dependencies:
+
+   ```sh
+   npm install
+   ```
+
+1. Authorize access to your org. Either Salesforce CLI or VS Code can be used for authorization and deployment. VS Code is recommended.
```
Comment (khawkins) [inline README.md:51]: Nit: Markdown is good about continuing ordered lists even when you specify the same ordinal repeatedly. But let's use increasing ordinals anyway, in this list.

File: force-app/main/default/lwc/viewAccountRecord/viewAccountRecord.js [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+import WEBSITE_FIELD from "@salesforce/schema/Account.Website";
+import INDUSTRY_FIELD from "@salesforce/schema/Account.Industry";
+import TYPE_FIELD from "@salesforce/schema/Account.Type";
+
 export default class ViewAccountRecord extends LightningElement {
   @api recordId;
   @api objectApiName;
 
-  @wire(getRecord, { recordId: "$recordId", layoutTypes: "Compact" })
+  fields = [NAME_FIELD, PHONE_FIELD, WEBSITE_FIELD, INDUSTRY_FIELD, TYPE_FIELD];
```
Comment (dbreese) [inline force-app/main/default/lwc/viewAccountRecord/viewAccountRecord.js:13]: Declare this array which is needed by the lightning-record-form's `fields` parameter.

Unfortunately, can't also use this in the @wire -- node 16 fails to compile, so tests will not run, so we have to re-declare a different array below.

**Review by AdamLiechty — APPROVED** [review]

**Review by khawkins — APPROVED** [review]

**Pull request merged**